app/graphql/types/customers/object.rb

```diff
@@ -41,6 +41,8 @@ class Object < Types::BaseObject
       field :provider_customer, Types::PaymentProviderCustomers::Provider, null: true
       field :subscriptions, [Types::Subscriptions::Object], resolver: Resolvers::Customers::SubscriptionsResolver
 
+      field :integration_customer, Types::IntegrationCustomers::Object, null: true
```

Comment:
I am not 100% sure, but I think that we should use `netsuite_customer` here, since netsuite customer has `has_one` relation